communitiesuk/funding-service-design-frontend
# FS-4071: Update contact page to meet new design

### Change description

- Use new contact_us_banner from the fund-store on the round model
- Update translations with Welsh equivalents

---


### How to test

- This will need to be deployed after https://github.com/communitiesuk/funding-service-design-fund-store/pull/290 as it depends on it
- Once both this and https://github.com/communitiesuk/funding-service-design-fund-store/pull/290 are deployed
- You can access the http://localhost:3008/contact_us?fund=COF&round=R1 on any environment
- Change the fund between COF/CYP/NSTF etc.  and observe differences
- We don't have Welsh atm, but this will be done in https://dluhcdigital.atlassian.net/browse/FS-4327
